Source/Extensions/Blazorise.DataGrid/DataGrid.razor

```diff
@@ -24,7 +24,7 @@
                         }
                         else
                         {
-                            <TableHeaderCell Style="cursor: pointer" Clicked="@(() => OnSortClicked( column ) )" width="@column.Width">
+                            <TableHeaderCell Style="@(Sortable ? "cursor: pointer" : "")" Clicked="@(() => OnSortClicked( column ) )" width="@column.Width">
```

Comment:
I would also add `Cursor` attribute to this one.

Reply:
Will do!